### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by DrewBradley** [DrewBradley/travel-tracker#32]
review accessibilty breakdown on Lighthouse and Wave:
Lighthouse:
- [x] Heading elements are not sequential
- [x] Form elements missing associated labels

Wave:
- [x] MISSING FORM LABELS
- [x] empty headers
- [x] orphaned labels!!!
- [x] skipped heading level

Other:
- [x]  ARIA labels
- [x]  tab-index
